templates/partial/_microcloud-navigation.html

```diff
@@ -0,0 +1,43 @@
+<div id="g-navigation" class="p-navigation is-dark">
+  <div class="p-navigation__row global-nav-row">
+    <div class="p-navigation__banner">
+      <a class="p-navigation__item p-global-nav-anchors" href="/">Canonical</a>
+      <a href="#g-navigation" class="p-navigation__toggle--open p-global-nav-anchors" title="menu">Menu</a>
+      <a href="#g-navigation-closed" class="p-navigation__toggle--close p-global-nav-anchors" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav">
+      <ul class="p-navigation__items"></ul>
+      <ul class="p-navigation__items global-nav"></ul>
+    </nav>
+  </div>
+</div>
+<header id="navigation" class="p-navigation is-dark" style="z-index: 9;">
+  <div class="row microsite-menu">
+    <div class="p-navigation__banner col-3">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/microcloud">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95" />
```

Comment:
`image_template`? Do you have any issue with it?

Reply:
I've seen issues with the image_template in the last couple of months, yes, but that's not why I chose not to use it. I just repeated what we're already doing for the [data subnav](https://github.com/canonical/canonical.com/blob/424d172bd4bb198bb2328154b4cd50b4073c7028/templates/partial/_data-navigation.html#L20)